Added note for escaping LDAP password in template.env

C0d3rZ3r0 opened the pull request:
If the LDAP password contains special characters that need escaping, they must be doubly escaped inside .env.

Added a reminder against the password field for that.

qh4t commented:
Sorry, I just noticed that this project has no `LICENSE` (#65) so I'm closing this PR.

C0d3rZ3r0 committed:
Added note for escaping LDAP password in template.env

If the LDAP password contains special characters that need escaping, they must be doubly escaped inside .env.

Added a reminder against the password field for that.
